pom.xml

```diff
@@ -8,6 +8,13 @@
     <artifactId>api</artifactId>
     <version>4.0.0-SNAPSHOT</version>
 
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>https://opensource.org/licenses/mit-license.php</url>
```

Comment:
Examples I've seen have shown that for the licenses block, they point to the central definition of the license. The included license in the repo is considered the master license for the project. We do not need the licenses stanza to be compliant, however - it's just extra.

Do we have a central posted license, outside of our repos, that we can link to?